src/brsTypes/components/RoSGNode.ts

```diff
@@ -1550,6 +1610,40 @@ export class RoSGNode extends BrsComponent implements BrsValue, BrsIterable {
         },
     });
 
+    /* Checks whether the subtype of the subject node is a descendant of the subtype nodeType
+     * in the SceneGraph node class hierarchy.
+     *
+     *
+     */
+    private issubtype = new Callable("issubtype", {
+        signature: {
+            args: [new StdlibArgument("nodeType", ValueKind.String)],
+            returns: ValueKind.Boolean,
+        },
+        impl: (interpreter: Interpreter, nodeType: BrsString) => {
+            return BrsBoolean.from(isSubtypeCheck(this.nodeSubtype, nodeType.value));
+        },
+    });
+
+    /* Checks whether the subtype of the subject node is a descendant of the subtype nodeType
+     * in the SceneGraph node class hierarchy.
+     *
+     *
+     */
+    private parentsubtype = new Callable("parentsubtype", {
+        signature: {
+            args: [new StdlibArgument("nodeType", ValueKind.String)],
+            returns: ValueKind.String | ValueKind.Invalid,
```

Comment:
Unfortunately the type system in this project isn't as expressive as TypeScript's, so this likely doesn't do what you expect.  If this function can return a string or `invalid`, the output type will have to be `ValuKind.Object`.  The reference docs for `parentSubtype` probably have a typo since they don't mention returning `invalid` at all.

```suggestion
            returns: ValueKind.Object,
```